NickvisionMoney.Shared/Controllers/AccountViewController.cs

```diff
@@ -410,19 +410,31 @@ private int SortTransactions(uint a, uint b)
         }
         else if (SortTransactionsBy == SortBy.Date)
         {
-            compareTo = _account.Transactions[a].Date.CompareTo(_account.Transactions[b].Date);
+            compareTo = GetCompareToByDate();
```

Comment:
Instead of making this a separate method just for this how about you turn the body of this else if into:
```csharp
compareTo = _account.Transactions[a].Date.CompareTo(_account.Transactions[b].Date);
compareTo = compareTo == 0 ? a.CompareTo(b) : compareTo;
```